programs/monaco_protocol/src/instructions/matching/on_order_creation.rs

```diff
@@ -122,6 +122,7 @@ pub fn on_order_creation(
                     order_match.stake,
                 )
                 .map_err(|_| CoreError::MatchingRemainingLiquidityTooSmall)?;
+            market_liquidities.update_stake_matched_total(order_match.stake)?;
```

Comment:
A test that this gets called would be great 👍

Reply:
done